ex12_ch3.py

```diff
@@ -0,0 +1,25 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+"""
+Created on Thu May 10 15:34:22 2018
+
+@author: amyhaddad
+"""
+
+#step1: get input for principal amount
+principal_amount = int(input("Enter the principal amount: $"))
+
+#step2: get input for interest rate as a percent (ie, "15" and not ".15")
+interest_rate = float(input("Enter the interest rate: "))
```

Comment:
`float` is probably better because interest rates can be decimals, such as 15.5%.